packages/js/e2e-core-tests/src/specs/merchant/wp-admin-product-new.test.js

```diff
@@ -213,10 +241,172 @@ const runAddVariableProductTest = () => {
 			await expect(page).toFill('input[name="variable_width[2]"]', '20');
 			await expect(page).toFill('input[name="variable_height[2]"]', '15');
 
-			await page.focus('button.save-variation-changes');
-			await expect(page).toClick('button.save-variation-changes', {text: 'Save changes'});
-			// @todo: https://github.com/woocommerce/woocommerce/issues/30580
+			await saveChanges();
 		});
+
+		it( 'can bulk-edit variations', async () => {
+			// Verify that all 'Downloadable' checkboxes are UNCHECKED.
+			await expandVariations();
+			for ( let i = 0; i < 8; i++ ) {
+				const chkbox = await page.$(
+					`input[name="variable_is_downloadable[${ i }]"]`
+				);
+				const isChecked = await (
+					await chkbox.getProperty( 'checked' )
+				 ).jsonValue();
+				expect( isChecked ).toEqual( false );
+			}
+
+			// Perform the 'Toggle "Downloadable"' bulk action
+			await selectVariationAction('Toggle "Downloadable"');
+
+			await clickGoButton();
+			await uiUnblocked();
+			await uiUnblocked();
```

Comment:
Did you intend to call `await uiUnblocked();` twice here?  It works, but I wasn't sure if it was intentional.

Reply:
Hi @lanej0, thank you very much for reviewing it 😄 

> Did you intend to call await uiUnblocked(); twice here?

Great that you noticed it. Yes, I really wanted to call it twice because as you would notice from the screen recording below, the loading spinner appeared twice with only a very small gap between them:

https://user-images.githubusercontent.com/4509348/148747906-97d200bf-61f0-47bf-b91c-a13354d114ea.mov

That's why I decided to call `uiUnblocked()` twice. 
I also tried using just one call, and it worked out fine as well. I guess that's because sometimes the gap was too small enough for the test to notice it. But I still went with the double call to surely avoid flakiness. let me know what you think. If you're good with this approach I will go ahead and merge it.